server/mqtt.go

```diff
@@ -1090,7 +1102,7 @@ func (s *Server) mqttCreateAccountSessionManager(acc *Account, quitCh chan struc
 	c := s.createInternalAccountClient()
 	c.acc = acc
 
-	id := getHash(s.Name())
+	id := s.NodeName()
```

Comment:
This is synonymous; took the liberty of putting the nit here.

Reply:
ok